docs/workgroup/charter.rst

```diff
@@ -0,0 +1,136 @@
+===================================
+CPython Documentation Working Group
+===================================
+
+Purpose & Common Goals
+----------------------
+
+This workgroup will support efforts to improve
+and maintain CPython's documentation through curation,
+internationalization, and special projects. Its focus will also include
+tutorials, how-to documents, the devguide and other documentation in support
+of the PSF's mission to expand the global Python userbase.
+Quality and effectiveness will be key goals as we build a
+diverse team of community members, documentarians, and core developers.
```

Comment:
Is "documentarians" a term of art? Never heard of it before, and it feels sort of made up!

Reply:
Write the Docs [has a dedicated page](https://www.writethedocs.org/documentarians/) for defining this term.

[According to dictionary.com](https://www.dictionary.com/browse/documentarian), the word has a meaning that somewhat fits our usecase.

> documentarian
> [ dok-yuh-men-tair-ee-uhn, -muhn- ]
> noun
> Movies, Television. a filmmaker, producer, etc., who specializes in [documentaries](https://www.dictionary.com> /browse/documentary).
> a writer, photographer, or other artist whose work constitutes a document or [documentary](https://www.dictionary.com/browse/documentary) of an aspect of life.

Reply:
In addition to echoing what @pradyunsg , I've heard it commonly used around the Python documentation sphere, yes.

Reply:
Sounds good, thanks for the explanation.

A

Reply:
One more reason to give us kind of permissions on here (at least triage...)